src/schemas/json/workflows.json

```
@@ -259,6 +259,48 @@
         },
         "for": {
           "$ref": "#/definitions/for"
+        },
+        "parallel": {
+          "type": "object",
+          "description": "Run branches or iterations in parallel",
+          "additionalProperties": false,
+          "properties": {
+            "exception_policy": {
+              "description": "determines action of other branches when an exception occurs",
```

```suggestion
              "description": "An action of other branches when an exception occurs",
```

Optional fix: sentence start consistency violated: `An action` may sound better.